packages/oc-get-unix-utc-timestamp/README.md

```diff
@@ -0,0 +1 @@
+# oc-get-unix-utc-timestamp
```

Comment:
Again, just a little description. Also, can we remove 'oc' from the name? I think this is more of a generic "get unix utc timestamp" module.